external-declarations/src/code-mod/fixer/code-fixer-applier.ts

```diff
@@ -31,20 +32,20 @@ export async function fixProjectRaw(
     documentRegistry: DocumentRegistry | undefined,
     snapShotRegistry: VersionedFileRegistry,
     fixableErrors: Set<number>,
-    selectFix: (service: LanguageService, diag: Diagnostic, fixes: readonly CodeFixAction[], files: VersionedFileRegistry, signal: BasicAbortSignal) => Promise<number | undefined>,
+    userPreferences: ts.UserPreferences,
+    selectFix: (service: LanguageService, diag: Diagnostic, fixes: readonly CodeFixAction[], files: VersionedFileRegistry, signal: BasicAbortSignal) => Promise<number | undefined> | (number | undefined),
     validateFix?: (service: LanguageService) => Promise<boolean>,
     onProjectLoaded?: (service: LanguageService, documentRegistry: DocumentRegistry, files: VersionedFileRegistry, signal: BasicAbortSignal) => Promise<void>,
 ) {
     documentRegistry = documentRegistry ?? createDocumentRegistry(
         host.useCaseSensitiveFileNames?.(),
         host.getCurrentDirectory(),
-    )
+    );
     const service = ts.createLanguageService(host, documentRegistry);
-
     const program = service.getProgram()!;
     const signal: BasicAbortSignal = { isAborted: false };
 
-    await onProjectLoaded?.(service, documentRegistry, snapShotRegistry, signal);
+    onProjectLoaded && (await onProjectLoaded?.(service, documentRegistry, snapShotRegistry, signal));

```

Comment:
Can you drop the optional chaining here?

Reply:
Yes. dropped